src/node/src/common.js

```diff
@@ -43,21 +43,20 @@ var _ = require('lodash');
 
 /**
  * Get a function that deserializes a specific type of protobuf.
- * @param {function()} cls The constructor of the message type to deserialize
- * @param {bool=} binaryAsBase64 Deserialize bytes fields as base64 strings
- *     instead of Buffers. Defaults to false
+ * @param {function()} cls The ProtoBuf.Type of the message type to deserialize
+ * @param {bool=} enumsAsStrings Deserialize enum fields as key strings
+ *     instead of numbers. Defaults to false
  * @param {bool=} longsAsStrings Deserialize long values as strings instead of
  *     objects. Defaults to true
  * @return {function(Buffer):cls} The deserialization function
  */
-exports.deserializeCls = function deserializeCls(cls, binaryAsBase64,
+exports.deserializeCls = function deserializeCls(cls, enumsAsStrings,
                                                  longsAsStrings) {
-  if (binaryAsBase64 === undefined || binaryAsBase64 === null) {
-    binaryAsBase64 = false;
```

Comment:
This options isn't implemented, yet, but now that the base64 stuff is in again, I can certainly add it back.